<launch>

	<node 

		name = "serial_node"
		pkg = "rosserial_python"
		type = "serial_node.py"
		respawn = "true" > 

		<param 
			name = "port"
			type = "string"
			value = "/dev/ttyACM0" />

		<param
			name = "baud"
			type = "int"
			value = "9600" />
	</node>


	<node
		name = "arduinoListener"
		pkg = "drone_development"
		type = "arduinoListener.py"
		respawn = "true" >
	</node>


	<node

		pkg = "terarangerone"
		type = "terarangerone_node"
		name = "terarangerone"
		respawn = "true" > 

	</node>

	<node

		pkg = "terarangerone2"
		type = "terarangerone_node2"
		name = "terarangerone2"
		respawn = "true" > 

	</node>

	<node

		pkg = "terarangerone3"
		type = "terarangerone_node3"
		name = "terarangerone3"
		respawn = "true" > 

	</node>

	<node

		pkg = "terarangerone4"
		type = "terarangerone_node4"
		name = "terarangerone4"
		respawn = "true" > 

	</node>


	<node 
		name="teraListener" 
		pkg = "drone_development" 
		type="terarangerListener.py"
		respawn = "true" > 
	</node>

	<node 
		name = "mainCode" 
		pkg = "drone_development" 
		type="main_code.py"
		output = 'screen' > 
	</node>


	

</launch>
		

